src/cli/build/plugin.js

```diff
@@ -184,7 +184,11 @@ let state = {
     // TODO: When we make the postcss plugin async-capable this can become async
     let files = fastGlob.sync(this.contentPatterns.all)
 
+    let checkBroadPattern = createBroadPatternCheck(this.contentPatterns.all)
```

Comment:
Descriptive, but horrible haha. Any thoughts on a better name? 🤔 

Idea is to check the globs once to see if any of the globs are a potential issue. Then the function returned will check every single file to see if it matches on of the globs _and_ is located in `node_modules`.

/cc @philipp-spiess

Reply:
hahah I think it's reasonable yeah. No better idea for a name right now either 😬